puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/clone_foreman.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -ex
+
+rm -rf foreman/
+git clone https://github.com/theforeman/foreman --branch "${foreman_branch}"
```

Comment:
Blast - I pushed it to a branch instead. So I just opened a PR with it instead -- https://github.com/theforeman/foreman-infra/pull/203